src/app/static/src/components/parameters/EditPhases.vue

```diff
@@ -314,6 +342,14 @@ export default defineComponent({
         position: absolute;
         padding: 0.5rem;
 
+        .disable-select {
+          user-select: none; /* supported by Chrome and Opera */
+          -webkit-user-select: none; /* Safari */
+          -khtml-user-select: none; /* Konqueror HTML */
+          -moz-user-select: none; /* Firefox */
+          -ms-user-select: none; /* Internet Explorer/Edge */
+        }
```

Comment:
Any more of this and we'll need a [prefix mixin](https://sass-lang.com/documentation/style-rules/declarations#interpolation) ;)

Reply:
Nice, will add that if it crops up again